src/main/kotlin/io/bazel/kotlin/compiler/BazelK2JVMCompiler.kt

```diff
@@ -22,8 +22,10 @@ import org.jetbrains.kotlin.cli.jvm.K2JVMCompiler
 import org.jetbrains.kotlin.config.Services
 
 @Suppress("unused")
-class BazelK2JVMCompiler(private val delegate: K2JVMCompiler = K2JVMCompiler()) {
-  fun exec(errStream: java.io.PrintStream, vararg args: kotlin.String): ExitCode {
+class BazelK2JVMCompiler {
+  fun exec(errStream: java.io.PrintStream, vararg args: String): ExitCode {
+    System.setProperty("zip.handler.uses.crc.instead.of.timestamp", "true")
```

Comment:
This is painful (that it's set as a global property in an exec() method I mean, nothing to be done about that if we want to ensure it, I suspect)

Reply:
It also doesn't work. Will remove.